pkg/trtl/config/replica_test.go

```diff
@@ -0,0 +1,213 @@
+package config_test
+
+import (
+	"errors"
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/stretchr/testify/require"
+	"github.com/trisacrypto/directory/pkg/trtl/config"
+)
+
+func TestReplicaConfigureNoModification(t *testing.T) {
+	s1 := makeRandomTestStrategy()
+	s2 := makeRandomTestStrategy()
+
+	conf := config.ReplicaConfig{
+		Enabled:        true,
+		PID:            53,
+		Region:         "ibelin",
+		Name:           "balin",
```

Comment:
i think it's "balian" 😉

Reply:
Hehe it definitely is - darn you cold fog brain!